### PR TITLE
Add 'publish-hyper' community supported example

### PR DESCRIPTION
**Pull request by tsjoblad**

### CODE REVIEW

File: Community-Supported/publish-hyper/publish-hyper-file.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    TableName, \
+    HyperException
+
+
+# Configure for TSC to publish
+# Note: Do not store creds/tokens in plaintext, please use env vars :)
+hyper_name = 'customer.hyper'
+server_address = 'https://10ax.online.tableau.com/'
+site_name = 'mysitename'
+project_name = 'projectname'
```
Comment (jonas-eckhardt) [inline Community-Supported/publish-hyper/publish-hyper-file.py:17]: Minor: you call the first one my.. the other one just project... maybe call mysite and myproject?

File: Community-Supported/publish-hyper/publish-hyper-file.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+token_name = 'tokennamehere'
+token_value = 'tokenherebutpleaseconsidersecuritypolicies'
```
Comment (jonas-eckhardt) [inline Community-Supported/publish-hyper/publish-hyper-file.py:19]: Can you add a comment about what the token is and where to get it from?

File: Community-Supported/publish-hyper/publish-hyper-file.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    NOT_NULLABLE, NULLABLE, SqlType, TableDefinition, \
+    Inserter, \
+    escape_name, escape_string_literal, \
+    TableName, \
+    HyperException
+
+
+# Configure for TSC to publish
+# Note: Do not store creds/tokens in plaintext, please use env vars :)
+hyper_name = 'customer.hyper'
```
Comment (jonas-eckhardt) [inline Community-Supported/publish-hyper/publish-hyper-file.py:14]: can we call this variable hyper_file_path?

File: Community-Supported/publish-hyper/publish-hyper-file.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    columns=[
+        TableDefinition.Column(name='Customer ID', type=SqlType.text(), nullability=NOT_NULLABLE),
+        TableDefinition.Column(name='Customer Name', type=SqlType.text(), nullability=NOT_NULLABLE),
+        TableDefinition.Column(name='Loyalty Reward Points', type=SqlType.big_int(), nullability=NOT_NULLABLE),
+        TableDefinition.Column(name='Segment', type=SqlType.text(), nullability=NOT_NULLABLE)
+    ]
+)
+
+
+def insert_data():
```
Comment (jonas-eckhardt) [inline Community-Supported/publish-hyper/publish-hyper-file.py:36]: Can you add a comment + link that this function is taken from the single table creation example?

File: Community-Supported/publish-hyper/README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# publish-hyper
```
Comment (jonas-eckhardt) [inline Community-Supported/publish-hyper/README.md:1]: Don't you also need to add a link + short description to this example to the high-level README?

File: Community-Supported/publish-hyper/README.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+## __Publishing a Single-Table Hyper File Directly to Tableau Online/Server__
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+This sample demonstrates how to leverage the Hyper API and Tableau Server Client Library to do the following:
+- Create a single-table Hyper file
+- Publish the file as a datasource directly to Online/Server, without a .tdsx file
```
[the remark this answers is not in the document]
Comment (d45) [reply Community-Supported/publish-hyper/README.md:8]: Yes. It's better to list them both.

File: Community-Supported/publish-hyper/README.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# publish-hyper
+## __Publishing a Single-Table Hyper File Directly to Tableau Online/Server__
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+This sample demonstrates how to leverage the Hyper API and Tableau Server Client Library to do the following:
+- Create a single-table Hyper file
```
[the remark this answers is not in the document]
Comment (d45) [reply Community-Supported/publish-hyper/README.md:7]: You can tag the file name extensions with the markdown for code (`)  as in `.hyper` file

File: Community-Supported/publish-hyper/publish-hyper-file.py (hunk 2 above)
Comment (d45) [reply Community-Supported/publish-hyper/publish-hyper-file.py:19]: You could send them to here for a start [Personal Access Tokens](https://help.tableau.com/current/server/en-us/security_personal_access_tokens.htm)

File: Community-Supported/publish-hyper/publish-hyper-file.py [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+        
+        print(f"Publishing {hyper_name} to {project_name}...")
+        # Publish datasource
+        datasource = server.datasources.publish(datasource, path_to_database, publish_mode)
+        print("Datasource published. Datasource ID: {0}".format(datasource.id))
+
+
+if __name__ == '__main__':
+    insert_data()
+    publish_hyper()
```
Comment (d45) [inline Community-Supported/publish-hyper/publish-hyper-file.py:109]: Might want to get rid of the "No newline at end of file" warning.

**Comment by d45** [comment]
@tsjoblad @jonas-eckhardt  
Hi Tucker, are you planning to merge this PR?   I want to point to the sample from the Hyper API docs.  Thanks!

**Comment by tsjoblad** [comment]
> Hi Tucker, are you planning to merge this PR? I want to point to the sample from the Hyper API docs. Thanks!
@d45 yes--just back from time-off, will do tonight and ping you. Thanks for the bump!

**Comment by d45** [comment]
Cool! Thanks, Tudker! (@tsjoblad )  I'll check the path to the sample tonight. We have the Hyper API release 11 tomorrow.

**Pull request merged**